packages/core/src/actors/callback.ts

```diff
@@ -24,10 +24,10 @@ export function fromCallback<TEvent extends EventObject>(
   invokeCallback: InvokeCallback
 ): ActorBehavior<TEvent, undefined> {
   const behavior: ActorBehavior<TEvent, undefined, CallbackInternalState> = {
-    start: (state, { self }) => {
-      self.send({ type: startSignalType } as TEvent);
+    start: (_state, actorCtx) => {
+      actorCtx.self.send({ type: startSignalType } as TEvent);
 
-      return state;
+      return behavior.getInitialState(actorCtx);
```

Comment:
as per the discussion in other comments - this return value should likely be removed